packages/runner/lib/exec.ts

```diff
@@ -80,7 +80,11 @@ export async function exec(
                     throw new Error(`Default exports is not a function but a ${typeof scriptExports.default}`);
                 }
                 if (isAction) {
-                    return await scriptExports.default(nango, codeParams);
```

Comment:
even if the input in the request was empty we were sending an empty object which was confusing and makes for a bad experience for a user of an action IMO